src/pages/BooksDetail.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+import book from "../assets/team.svg";
+import { motion } from "framer-motion";
+import AnimateButton from "../helpers/AnimateButton";
+import ContactModal from "../components/ContactModal";
+import EditBookModal from "../components/EditBookModal";
+import EditImageModal from "../components/EditImageModal";
+
+function BooksDetail() {
```

Comment:
The data of the book, where does it come from? Is it fetch or how? Can we prepare something so that it is easier to update the code later on.

Reply:
I'm not sure abou this one. I think we are getting the data from firebase for each book. But I have done some changes.